pgcli/pgexecute.py

```diff
@@ -186,7 +185,9 @@ def connect(self, database=None, user=None, password=None, host=None,
 
             cursor = conn.cursor()
 
-        superuser = self._select_one(cursor, "select current_setting('is_superuser')::bool")[0]
+        cursor.execute("SHOW ALL")
```

Comment:
My thinking was, if we're going to get the database settings, we might as well just get them all and get it over with, in the future maybe somebody wants to examine some other setting and it'll already be there.

Yes it does actually throw the error `must be superuser to examine "is_superuser"` on redshift, which unfortunately means that the current version on pypi is broken for redshift users.